.taskcluster.yml

```diff
@@ -0,0 +1,63 @@
+version: 0
+allowPullRequests: public
+tasks:
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events:
+          - push
+          - release
+    scopes:
+        - secrets:get:project/taskcluster/testing/taskcluster-stats-collector
+    payload:
+      env:
+        NO_TEST_SKIP: true
+      features:
+        taskclusterProxy: true
+      maxRunTime: 3600
+      image: node:8
+      command:
+        - /bin/bash
+        - '--login'
+        - '-c'
+        - >-
+          git clone {{event.head.repo.url}} repo && cd repo && git config
+          advice.detachedHead false && git checkout {{event.head.sha}} &&
+          yarn &&
+          yarn test
+    metadata:
+      name: taskcluster-stats-collector tests
+      description: Run tests for taskcluster-stats-collector in Taskcluster
+      owner: '{{ event.head.user.email }}'
+      source: '{{ event.head.repo.url }}'
+
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events:
+          - pull_request.opened
+          - pull_request.reopened
+          - pull_request.synchronize
+    payload:
+      features:
+        taskclusterProxy: true
+      maxRunTime: 3600
+      image: node:8
+      command:
+        - /bin/bash
+        - '--login'
+        - '-c'
+        - >-
+          git clone {{event.head.repo.url}} repo && cd repo && git config
+          advice.detachedHead false && git checkout {{event.head.sha}} &&
+          yarn &&
+          unset TASK_ID &&
```

Comment:
Should we consider having `Secrets` turn itself on given some other value than `TASK_ID`? Perhaps setting things explicitly would be nicer.